src/main/java/com/k2view/agent/httpsender/oauth/OAuthRequestBuilder.java

```diff
@@ -69,4 +69,4 @@ String buildPostData() {
         }
         return postStr;
     }
-}
```

Comment:
?

Reply:
i removed a white space 

look at the line below
@idopolak